core/tee/tee_svc.c

```diff
@@ -43,6 +42,12 @@
 #include <trace.h>
 #include <kernel/trace_ta.h>
 #include <kernel/chip_services.h>
+#include <kernel/misc.h>
+
+#define MAX_BASE64_BUFFER_SIZE 120
+
+extern int base64_encode(const unsigned char *in,  unsigned long len,
```

Comment:
I don't think base64 is useful in this case. We can just return the "ext.tee.arm.genericTimerInfo" property as a binary block.